### DISCUSSION TITLE
Should we remove the PresentationFramework.Luna.dll

### DISCUSSION THREAD

**Discussion by lindexi** [dotnet/wpf#6711] [General]
As we all know, the PresentationFramework.Luna.dll is the Windows XP theme assembly. But we no longer support Windows XP.

Should we remove the PresentationFramework.Luna.dll to save the size?

- PresentationFramework.Luna.dll   643.60KB

Ref: https://github.com/dotnet/sdk/issues/3265

**Comment by AraHaan** [comment]
Luna, and Royale were both Windows XP themes with RoyaleNoir being unreleased but popular themes.

with Applications using both of the themes as they are still popular even on modern OS's for their production programs makes this an unlikely option for them.

Besides do these themes also include a built in way of replacing the NC area as well?

I do however wish there was an PresentationFramework.Office for WPF applications who want to use the visual styles from Office 2010.
